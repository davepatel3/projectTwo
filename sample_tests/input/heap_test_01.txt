2262
create_file
48
rRIFHZq2xzLsrhG8C
0
create_directory
NQ-t_d
0
get_available_size
create_file_root
8
4E_K
create_file
5
hkaUA-5uX2S20qwH
2
print_layout
create_directory
MQcZ-GRsbi2yz7 W-L
2
create_file
22
pdzeoGuyjXd
2
create_directory
L9DsTPwNy4v3dRDKxA
5
get_largest_file_handle
create_file
118
k6Q37Q
2
exist
2
create_file
11
TqAC4zn QF6ioN
2
create_file
115
2xOpY5ripBdaDEdNpVA
0
get_available_size
get_absolute_path
7
create_directory
Yys07gn
7
create_directory
XQfpKrLKP
0
create_file_root
21
91 E7waMfnL_FLFztm
create_file
84
Ki3HIcBmQcRRfl1GH
7
create_file
37
99DduUt
2
get_available_size
get_largest_file_handle
get_file_size
1
get_largest_file_handle
get_name
14
create_file
85
ksg 
2
get_available_size
create_file
6
b8rNh5tEdt
12
create_file
46
dYuaV9MDnFUH3BH
12
create_file
10
II6YO4ieYY7
11
get_handle
/NQ-t_d/pdzeoGuyjXd
create_directory
yAXtIBhfu6Pvh3
11
create_file
9
5hkgX-M
7
create_file
32
paAfjC
0
create_file
39
meof38h
2
create_directory
BfxkaPi-U
0
create_directory
d 4leuMcFaRdrTEWyT_E
7
create_directory
gorBk3DPtvQ8
20
get_largest_file_handle
print_layout
create_directory
 LQixcN-ZUbNW5iOB Q0
12
get_available_size
get_name
1
create_file
24
cKvKUVdrud
25
get_largest_file_handle
create_file
57
979LvJFbdS148D
26
create_directory
bbdXFp-NBznLfUceWK
11
create_directory
YHSHmT
27
create_directory
V4ANogU5m7i
11
create_file
86
jbGwoEsY
24
get_name
4
create_file
69
XCsrMjUVjBxM 
26
get_name
26
quit
