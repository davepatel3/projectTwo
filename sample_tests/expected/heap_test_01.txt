create_file
1
create_directory
2
get_available_size
2214
create_file_root
3
create_file
4
print_layout
[F]rRIFHZq2xzLsrhG8C (size = 48)
[D]NQ-t_d
	[F]hkaUA-5uX2S20qwH (size = 5)
[F]4E_K (size = 8)

create_directory
5
create_file
6
create_directory
7
get_largest_file_handle
1
create_file
8
exist
1
create_file
9
create_file
10
get_available_size
1935
get_absolute_path
/NQ-t_d/MQcZ-GRsbi2yz7 W-L/L9DsTPwNy4v3dRDKxA
create_directory
11
create_directory
12
create_file_root
13
create_file
14
create_file
15
get_available_size
1793
get_largest_file_handle
8
get_file_size
48
get_largest_file_handle
8
get_name
Ki3HIcBmQcRRfl1GH
create_file
16
get_available_size
1708
create_file
17
create_file
18
create_file
19
get_handle
6
create_directory
20
create_file
21
create_file
22
create_file
23
create_directory
24
create_directory
25
create_directory
26
get_largest_file_handle
8
print_layout
[F]rRIFHZq2xzLsrhG8C (size = 48)
[D]NQ-t_d
	[F]hkaUA-5uX2S20qwH (size = 5)
	[D]MQcZ-GRsbi2yz7 W-L
		[D]L9DsTPwNy4v3dRDKxA
			[D]Yys07gn
				[F]II6YO4ieYY7 (size = 10)
				[D]yAXtIBhfu6Pvh3
					[D]gorBk3DPtvQ8
			[F]Ki3HIcBmQcRRfl1GH (size = 84)
			[F]5hkgX-M (size = 9)
			[D]d 4leuMcFaRdrTEWyT_E
	[F]pdzeoGuyjXd (size = 22)
	[F]k6Q37Q (size = 118)
	[F]TqAC4zn QF6ioN (size = 11)
	[F]99DduUt (size = 37)
	[F]ksg  (size = 85)
	[F]meof38h (size = 39)
[F]4E_K (size = 8)
[F]2xOpY5ripBdaDEdNpVA (size = 115)
[D]XQfpKrLKP
	[F]b8rNh5tEdt (size = 6)
	[F]dYuaV9MDnFUH3BH (size = 46)
[F]91 E7waMfnL_FLFztm (size = 21)
[F]paAfjC (size = 32)
[D]BfxkaPi-U

create_directory
27
get_available_size
1566
get_name
rRIFHZq2xzLsrhG8C
create_file
28
get_largest_file_handle
8
create_file
29
create_directory
30
create_directory
31
create_directory
32
create_file
33
get_name
hkaUA-5uX2S20qwH
create_file
34
get_name
gorBk3DPtvQ8
quit
